### ISSUE TITLE
Enhance wins JavaScript to assign alt text to quote icon

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#4113]
### Overview

To comply with Web Content Accessibility Guidelines (WCAG), we need to ensure that image alt text on the Wins page is meaningful and appropriate for each image. This issue addresses the need for alt text on the big quote icon in the Wins cards, which are generated using JavaScript.

### Action Items

- [x] Understand the role of these components of the wins page:
  - [x] The main Wins web page: [wins.html](https://github.com/hackforla/website/blob/gh-pages/pages/wins/wins.html), which uses Liquid to include the template for each win card: [wins-card-template.html](https://github.com/hackforla/website/blob/gh-pages/_includes/wins-page/wins-card-template.html)
  - [x] The JavaScript code [wins js](https://github.com/hackforla/website/blob/gh-pages/assets/js/wins.js) running in the Wins web page.pro
- [x]  In [wins-card-template.html](https://github.com/hackforla/website/blob/gh-pages/_includes/wins-page/wins-card-template.html), find the img element with class 'wins-card-big-quote', and add an empty alt attribute (alt='').
- [x] In [wins-overlay.html](https://github.com/hackforla/website/blob/gh-pages/_includes/wins-page/wins-overlay.html), find the img element with class 'wins-card-big-quote', and add an empty alt attribute (alt='').
- [x] In [wins.js](https://github.com/hackforla/website/blob/gh-pages/assets/js/wins.js), search for the text '.wins-card-big-quote' in order to locate the line of code which sets the "src" attribute for the big quote icon. Add code to set the "alt" attribute of the image equal to "Quote from name" where name is the name of the contributor. For example, the alt text for the big quote icon on Mary Smith's card should be "Quote from Mary Smith".
- [x] A change will also be required in wins.js in the function updateOverlay to set the alt text in the Overlay (when the user clicks "See More").  Here the alt text can be copied from the original element in the same manner as the src attribute is retrieved.  
- [x] View the wins page to test the code at mobile, tablet and full-screen widths. Also also follow the "See More" page to check the alt tag on the modal overlay.

### Resources/Instructions
- [Wins webpage](https://hackforla.org/wins/)
- [wins.html](https://github.com/hackforla/website/blob/gh-pages/pages/wins/wins.html)
- [wins js](https://github.com/hackforla/website/blob/gh-pages/assets/js/wins.js)
- [wins-card-template.html](https://github.com/hackforla/website/blob/gh-pages/_includes/wins-page/wins-card-template.html)
- This issue was created as a follow-up to https://github.com/hackforla/website/issues/2569. See that issue for accessibility resources.

**Comment by ExperimentsInHonesty** [comment]
- @roslynwythe  Is there are reason why it has the dependency https://github.com/hackforla/website/issues/4112? It seems like they could be worked on at the same time? Am I missing something?

**Comment by jdingeman** [comment]
Removing dependency and marking as ready for prioritization.

**Comment by KazushiR** [comment]
Availability: weekdays
ETA: 10/20

**Comment by jdingeman** [comment]
Hi @KazushiR - thank you for picking up the issue, however it is in the new issue approval column and has not been prioritized yet. Please only pick up issues out of the prioritized backlog.

**Comment by ExperimentsInHonesty** [comment]
@roslynwythe Good call on this.  I would have said it was decorative, but after reading your issue, I agree that if I were using a screen reader, it would be good to hear that the text that followed is a quote.  Is that how we think the experience will be? and will that be awkward, with all the other references to that person's name that we are adding into the alt text?

**Comment by roslynwythe** [comment]
@ExperimentsInHonesty Yes it is a bit verbose, but since the chrome screen reader (by default) won't read any of the text on the wins card, the alt text provides crucial context.  Also, since a descriptive page title is key for communicating the purpose of a page for screen reader users, I suggest we change the page title from "Wins by Hack for LA" to "Let's celebrate our contributors' wins".

**Comment by ExperimentsInHonesty** [comment]
@roslynwythe I am going to move your renaming the wins page recommendation to the dev/pm meeting agenda, so I can move this issue to the prioritized backlog.

**Comment by jch1013** [comment]
Availability: Tuesday and Thursday afternoons
ETA: 3/25 end of day

**Comment by jch1013** [comment]
@roslynwythe Hi! quick question... Is it possible that the steps for the wins-card-template.html file and the wins-overlay.html file should be switched (or maybe the both apply to wins-card-template)? I'm not seeing any img elements with the class "github-icon" in wins-overlay, but I can see one in wins-card-template. Thanks!

**Comment by roslynwythe** [comment]
> @roslynwythe Hi! quick question... Is it possible that the steps for the wins-card-template.html file and the wins-overlay.html file should be switched (or maybe the both apply to wins-card-template)? I'm not seeing any img elements with the class "github-icon" in wins-overlay, but I can see one in wins-card-template. Thanks!

Hi @jch1013 the instruction for wins-overlay.html was incorrect.  It should have referred to 'wins-card-big-quote', 'github-icon'.  I have fixed the Action Item and I apologize for the confusion.

**Comment by jch1013** [comment]
Blocker: having trouble adding alt tag to overlay
Updated ETA: 4/2
Availability: afternoons on 3/31, 4/1
